src/scss/mixins/_mixins.scss

```diff
@@ -60,9 +60,9 @@
 
 @mixin focus-ring($show-border: false) {
   outline: 0;
-  box-shadow: 0 0 $focus-ring-blur $focus-ring-width rgba(var(--tblr-primary-rgb), 0.25);
+  box-shadow: 0 0 $focus-ring-blur $focus-ring-width rgba(var(--#{$prefix}-primary-rgb), 0.25);
```

Comment:
Shouldn't that be `--#{$prefix}primary-rg` instead?

Reply:
oops sorry, my bad